### PR TITLE
applications: machine_learning: Fix USB nrfx stack overflow

### PR DESCRIPTION
**Pull request by MarekPieta**
Change fixes crash when USB cable is plugged in. The crash is caused by USB nrfx stack overflow.

Jira: NCSDK-13736

### CODE REVIEW

File: applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf [hunk 1]
```diff
@@ -179,6 +179,7 @@ CONFIG_LOG_BACKEND_UART=y
 # Enable USB CDC ACM
 
 CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_NRFX_WORK_QUEUE_STACK_SIZE=1200
```
Comment (pdunaj) [inline applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf:182]: Doesn't it need to be fixed in Zephyr so all apps/samples benefit? If the stack size is too small?

**Review by pdunaj — APPROVED** [review]

File: applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf (hunk 1 above)
Comment (pdunaj) [reply applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf:182]: note that this should only affect nrf53 if there is such a need unless we know it affects other platforms too.

Comment (pdunaj) [reply applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf:182]: @jfischer-no , what do you think?

Comment (jfischer-no) [reply applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf:182]: That surprises me very much. Let's not change it in upstream.

**Review by jfischer-no — APPROVED** [review]

**Comment by shantha-14** [comment]
@MarekPieta  No downstream job executed in CI ? Has this been tested locally at your desk ?

**Comment by MarekPieta** [comment]
Yes, I tested it locally on my desk. @jumi-nordic, could you check locally to ensure that issue is gone for you too?

**Comment by mbolivar-nordic** [comment]
>  @jumi-nordic, could you check locally to ensure that issue is gone for you too?

Not merging this one yet as this request is still open

**Comment by shantha-14** [comment]
@MarekPieta  I am not going to wait for @jumi-nordic I will get this merged, if he find a bug we can fix in another PR

**Pull request merged**